bundles/org.openhab.binding.unifi/README.md

```diff
@@ -113,12 +113,24 @@ The following table describes the `poePort` configuration parameters:
 
 The `site` information that is retrieved is available as these channels:
 
-| Channel ID      | Item Type | Description                          | Permissions |
-|-----------------|-----------|--------------------------------------|-------------|
-| totalClients    | Number    | Total number of clients connected    | Read        |
-| wirelessClients | Number    | Number of wireless clients connected | Read        |
-| wiredClients    | Number    | Number of wired clients connected    | Read        |
-| guestClients    | Number    | Number of guest clients connected    | Read        |
+| Channel ID            | Item Type | Description                                                            | Permissions |
+|-----------------------|-----------|------------------------------------------------------------------------|-------------|
+| totalClients          | Number    | Total number of clients connected                                      | Read        |
+| wirelessClients       | Number    | Number of wireless clients connected                                   | Read        |
+| wiredClients          | Number    | Number of wired clients connected                                      | Read        |
+| guestClients          | Number    | Number of guest clients connected                                      | Read        |
+| guestVoucher          | String    | Guest voucher for access through the guest portal                      | Read        |
+| guestVouchersGenerate | String    | Generate additional guest vouchers for access through the guest portal | Write       |
+
+The `guestVouchersGenerate` string channel is a command only channel that will trigger voucher creation.
+It has configuration parameters to tailor the vouchers created:
+
+* `voucherCount`: Number of vouchers to create (default 1)
```

Comment:
Maybe put the parameters in a table to be more consistent with the other tables, like for `poePort`.